coap/Kconfig

```diff
@@ -82,6 +82,30 @@ menu "CoAP Configuration"
 
             If this option is disabled, redundent CoAP TCP code is removed.
 
+    config COAP_OSCORE_SUPPORT
+        bool "Enable OSCORE support within CoAP"
+        default n
+        help
+            Enable OSCORE functionality for CoAP.
+
+            If this option is disabled, redundent CoAP OSCORE code is removed.
+
+    config COAP_OBSERVE_PERSIST
+        bool "Enable Server Observe Persist support within CoAP"
+        default n
+        help
+            Enable Server Observe Persist support for CoAP.
+
+            If this option is disabled, redundent CoAP Observe Persist code is removed.
+
+    config COAP_WEBSOCKETS
```

Comment:
Should we consider adding newly added config options to `sdkconfig.ci` file in the examples, so that they are part of the test coverage?

Reply:
It was not required up until now. But, you may create one, since we want these configs to only affect the CI build (`sdkconfig.ci`) and not the normal build (`sdkconfig.defaults*`).

Reply:
OK, will do.  Found a compiler complaint in the OSCORE code which I am fixing, as well as seeing something when doing WebSockets.  So, glad you raised the question!

Reply:
Done